packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -131,6 +131,7 @@ export class TokenDetectionController extends PollingControllerV1<
     };
 
     this.initialize();
+    this.setIntervalLength(this.config.interval);
```

Comment:
Does this fix this controller so it actually uses the passed interval? If so, it seems important to call out in the Changes section in the PR description.

Reply:
yes indeed!

Reply:
done. Added a changelog entry in the PR description.